wallet/wallet-cli-lib/src/commands/mod.rs

```diff
@@ -47,6 +48,9 @@ pub enum WalletCommand {
         /// File path
         wallet_path: PathBuf,
 
+        /// If true save the seed phrase to the DB for later retrieval
```

Comment:
If 'save', the seed-phrase will be saved in the wallet file. If 'do-no-save', the seed-phrase will only be printed on the screen. Not saving the seed-phrase can be seen as a security measure to ensure sufficient secrecy in case that seed-phrase is reused elsewhere if this wallet is compromised.

Reply:
changed